digits/model/tasks/torch_train.py

```diff
@@ -136,6 +135,8 @@ def task_arguments(self, resources, env):
                 '--save=%s' % self.job_dir,
                 '--snapshotPrefix=%s' % self.snapshot_prefix,
                 '--snapshotInterval=%s' % self.snapshot_interval,
+                '--mean=%s' % self.dataset.path(constants.MEAN_FILE_IMAGE),
+                '--labels=%s' % self.dataset.path(self.dataset.labels_file),
```

Comment:
these two lines were moved to lines 150-151
line 192 addresses the generic dataset case (where we need to create an image file out of the .protobuf from the dataset)